.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,9 @@
+blank_issues_enabled: false
```

Comment:
🎉 https://help.github.com/en/github/building-a-strong-community/configuring-issue-templates-for-your-repository